brix-core/src/main/java/org/brixcms/web/nodepage/BrixNodePageRequestHandler.java

```diff
@@ -143,16 +144,14 @@ public PageParameters getPageParameters() {
             }
         }
 
-        @Override
-        public boolean isNewPageInstance() {
-            return page == null;
-        }
```

Comment:
Is this related to Wicket 9.x ?

Reply:
yep - this function isn't existing anymore in wicket 9, so we cant override it